### ISSUE TITLE
Wrong attributes encoding

### ISSUE THREAD

**Issue by mlagutko** [Overbryd/exoml#1]
When a tag contains more then one attribute the `encode` method generates wrong XML.

Here is the sample test
```
  test "back and forth 2 attributes" do
    xml = "<tag foo=\"bar\" bar=\"foo\">some text</tag>"
    assert xml == Exoml.encode(Exoml.decode(xml))
  end
```

Here is the result:
```
 1) test back and forth (ExomlTest)
     test/exoml_test.exs:4
     Assertion with == failed
     code:  assert xml == Exoml.encode(Exoml.decode(xml))
     left:  "<tag foo=\"bar\">some text</tag>"
     right: "<tag foo=\"bar\" >some text</tag>"
     stacktrace:
       test/exoml_test.exs:6: (test)
```

To fix the issue it is necessary to remove the method: https://github.com/Overbryd/exoml/blob/master/lib/exoml/encoder.ex#L45-L47

**Comment by semarco** [comment]
hi @Overbryd 
run into the same issue.
the fork from @mlagutko does already have the fix.  @mlagutko any chance to see a PR?
thanks

**Comment by Overbryd** [comment]
@pixelvitamina @mlagutko thanks for looking into that issue.

I am working on a fix right now.

**Comment by Overbryd** [comment]
Published `0.0.3` today https://hex.pm/packages/exoml
